src/utils.ts

```diff
@@ -36,6 +36,21 @@ export function existsSync(path: string): boolean {
 }
 
 /**
+ * Checks asynchronously if a path exists on the disk.
+ */
+export function existsAsync(path: string): Promise<boolean> {
+    return new Promise((resolve, reject) => {
+        try {
+            fs.access(path, (err) => {
+                resolve(err ? false : true);
+            });
+        } catch (e) {
+            resolve(false);
```

Comment:
Maybe you can try sending null, an invalid path, or an {} as the path parameter to see if that throws an exception or returns an error in the callback...